src/micromamba/config.cpp

```diff
@@ -88,6 +137,336 @@ set_config_describe_command(CLI::App* subcom)
     });
 }
 
+
+void
+set_config_file_command(CLI::App* subcom)
+{
+    auto& config = Configuration::instance();
+
+    auto& file_path = config.insert(Configurable("config_set_file_path", fs::path())
+                                        .group("cli")
+                                        .description("File path to set configuration"),
+                                    true);
+    subcom->add_option("--file", file_path.set_cli_config(fs::path()), file_path.description());
```

Comment:
can we also handle `--system`, `--env` and `--home`? 

And the default value shoudl be `--home` -> `env::home_directory() / ".condarc"`

Reply:
Jawohl, I'm on it :)